src/pages/invite/[invitation].tsx

```diff
@@ -1,12 +1,39 @@
 import { useRouter } from 'next/router'
-import React from 'react'
+import React, { useState } from 'react'
+import { CenterWrapper } from '../login'
+import RFC from '../../components/RFC'
+import ServiceEntity from '../../components/ServiceEntity'
+import CreatePassword from '../../components/CreatePassword'
+import ExternalCredentials from '../../components/ExternalCredentials'
+import TermsConditions from '../../components/TermsConditions'
+
+export interface entityTypes {
+  name: string
+  id: string
+  twoFactor: boolean
+}
 
 const Invitation = () => {
+  const [step, setStep] = useState(5)
```

Comment:
```suggestion
  const [step, setStep] = useState(1)
```
La página inicia en el último paso.